sample-notificator-domain/src/main/groovy/life/qbic/business/notification/create/CreateNotification.groovy

```diff
@@ -15,156 +19,100 @@ import life.qbic.business.subscription.fetch.FetchSubscriberOutput
  * @since: 1.0.0
  */
 
-class CreateNotification implements CreateNotificationInput, FetchSubscriberOutput{
+class CreateNotification implements CreateNotificationInput {
+  
     private final CreateNotificationOutput output
-    private final FetchSubscriberInput fetchSubscriberInput
-    private Map<Subscriber, String> notificationPerSubscriber
+    private final FetchUpdatedDataSource dataSource
+    private List<NotificationContent> notifications
 
-    CreateNotification(FetchSubscriberDataSource dataSource, CreateNotificationOutput output){
+    CreateNotification(FetchUpdatedDataSource dataSource, CreateNotificationOutput output) {
         this.output = output
-        this.fetchSubscriberInput = new FetchSubscriber(dataSource, this)
+        this.dataSource = dataSource
     }
 
     @Override
     void createNotifications(String date) {
-        fetchSubscriberInput.fetchSubscriber(date)
-        //assume notificationPerSubscriber will be filled by the output method fetchedSubscribers of the FetchSubscriber use case
-        output.createdNotifications(notificationPerSubscriber)
+      LocalDate localDate = LocalDate.parse(date)
+      
+      try {
+          //1. get todays notifications
+          Map<String, Status> sampleToStatus = dataSource.getUpdatedSamplesForDay(localDate)
+          //2. get projects
+          Map<String,List<String>> projectsWithSamples = getProjectsWithSamples(sampleToStatus.keySet().asList())
```

Comment:
Do we need all samples per project?
Can we skip this part and directly count and associate the number of samples for a project?
Just one method called getSamplesPerStatus(String project, Status status) or so?